src/components/CentralContainer/Sidebar/SidebarTabs/CustomTabPanel.tsx

```diff
@@ -23,25 +26,40 @@ interface CustomTabPanelProps {
  * @param props.children
  * @param props.tabName
  * @param props.title
+ * @param props.titleButtons
  * @return
  */
-const CustomTabPanel = ({children, tabName, title}: CustomTabPanelProps) => {
+const CustomTabPanel = ({
+    children,
+    tabName,
+    title,
+    titleButtons,
+}: CustomTabPanelProps) => {
     return (
         <TabPanel
             className={"sidebar-tab-panel"}
             value={tabName}
         >
-            <DialogTitle className={"sidebar-tab-panel-title-container"}>
-                <Typography
-                    className={"sidebar-tab-panel-title"}
-                    level={"body-md"}
-                >
-                    {title}
-                </Typography>
-            </DialogTitle>
-            <DialogContent>
-                {children}
-            </DialogContent>
+            <Box className={"custom-tab-panel-container"}>
+                <DialogTitle className={"sidebar-tab-panel-title-container"}>
+                    <Typography
+                        className={"sidebar-tab-panel-title"}
```

Comment:
As we discussed, we want the title buttons to be right aligned. Let's add `flex-grow: 1;` into `.sidebar-tab-panel-title`.